src/styles/theme/colors.ts

```diff
@@ -0,0 +1,144 @@
+export const baseColors = {
+    black: "#000000",
+    white: {
+        pure: "#ffffff",
+        transparent: "rgba(255, 255, 255, 0.65)",
+    },
+    // named darkest to lightest
+    dark: {
+        one: "#141219",
+        two: "#19171f",
+        three: "#1e1b25",
+        four: "#25222e", // fka three
+        five: "#2f2a3c", // fka four
+        six: "#3b3649", // fka charcoal gray, its purplish...
```

Comment:
Can remove these comments, but in our *migration era* I think these are time savers when juggling old css vars and new color constants

Reply:
migration era ✨